zcash_client_sqlite/src/testing.rs

```diff
@@ -958,6 +959,105 @@ impl<Cache> TestState<Cache> {
         )
         .unwrap()
     }
+
+    /// Returns a vector of transaction summaries
+    pub(crate) fn get_tx_history(
+        &self,
+    ) -> Result<Vec<TransactionSummary<AccountId>>, SqliteClientError> {
+        let mut stmt = self.wallet().conn.prepare_cached(
+            "SELECT * 
+             FROM v_transactions 
+             ORDER BY mined_height, tx_index DESC",
```

Comment:
Sorry for the dumb question; haven't written much SQL since I was a teenager. You don't have to do `ORDER BY mined_height DESC, tx_index DESC` here?

Reply:
Good catch!

Reply:
Oh, that's actually the intentional ordering here to ascend on mined height and descend on tx_index.